### DISCUSSION TITLE
Guide for maps server-client

### DISCUSSION THREAD

**Discussion by Th3AnG3L** [azerothcore/azerothcore-wotlk#17868] [Guides & Tips]
Hello, can someone tell me guide information about this topics:
1. how to make from server side that i have download from azerothcore the latest map builds and i put them in server folder - how i can copy this to client side to update them?
2. how to update from server side to client side the items?

Additional what else i must do to transfer from server side to client side to be equal everything.

**Comment by virgo77** [comment]
If your client is a real WotLK 3.3.5a v12340, so you don't need to "copy" side by side.
The server's maps are built on the latest 3.3.5a 12340 client. So, if you follow the AC Tutos, you will get all you need to run a server and play with. The only thing you have to do is to check the maps versions used by AzerothCore as sometimes they are updated. So, go check this before running you core update.

No need to update from server to client, as the update system (if you mean patches) is up-to-date. So all the datas between server and client are compatible.

If you meant something else, so be more explicit. :)

Play well!

**Reply by Th3AnG3L** [reply]
i have a client 3.3.5a that is long time downloaded. I have noticed that from azeroth has updated maps lets say this year so i have downloaded it and my question is when i download this new maps how i can be sure that client side is the same equal stuff like it is in server side?
If its not then i need to make an patch or some guide would be nice so client side to be equal to server.
